app/wama.web/Controllers/UserToolController.cs

```diff
@@ -243,6 +243,7 @@ public IActionResult AddCertification(string memberId)
         public async Task<IActionResult> DeleteCertification(string memberId)
         {
             var account = await GetUserAccountAsync(memberId);
```

Comment:
The idea behind throwing this Exception is that the method isn't implemented. Hence no code should be in there, except  the exception you're throwing.